radio/src/tasks.cpp

```diff
@@ -86,11 +86,36 @@ void sendSynchronousPulses(uint8_t runMask)
 constexpr uint8_t MIXER_FREQUENT_ACTIONS_PERIOD = 5 /*ms*/;
 constexpr uint8_t MIXER_MAX_PERIOD = MAX_REFRESH_RATE / 1000 /*ms*/;
 
+#if defined(DEBUG)
+static uint16_t dbg_counter{0};
+#endif
+
 void execMixerFrequentActions()
 {
 #if defined(SBUS_TRAINER)
   // SBUS trainer
-  processSbusInput();
+    if (hasSerialMode(UART_MODE_SBUS_TRAINER) >= 0) {
+        processSbusInput();
+    }
+    else if (hasSerialMode(UART_MODE_IBUS_TRAINER) >= 0) {
+        processIbusInput();
+    }
+    else if (hasSerialMode(UART_MODE_CRSF_TRAINER) >= 0) {
+        processCrsfInput();
+    }
+    else if (hasSerialMode(UART_MODE_SUMD_TRAINER) >= 0) {
+        processSumdInput();
+    }
```

Comment:
Same here: when using software timers, only the relevant function will be periodically called.

Reply:
Yes, the next improvement of this PR will use soft-timers.